website/docs/r/datadog_tagrules.html.markdown

```diff
@@ -0,0 +1,100 @@
+---
+subcategory: "Datadog"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_datadog_monitor_tagrules"
+description: |-
+  Manages TagRules on the datadog Monitor.
+---
+
+# azurerm_datadog_monitor_tagrules
```

Comment:
Can we rename this to be `azurerm_datadog_monitor_tag_rule`?

Reply:
Since tagrules is a single input file, will the split be neccassary?

Reply:
@vikotha At least the last `s` can be removed as we try to make the resource name as a singular.

Reply:
Updated